test/symmetric_positive_definite.jl

```diff
@@ -8,14 +8,15 @@ types = [ Matrix{Float32},
         Matrix{Float64},
         MMatrix{3,3,Float32},
         # linear algebra in StataicArrays is a little too inaccurate for MMatrix{3,3,Float64}
-        # MMatrix{3,3,Float64}
+        MMatrix{3,3,Float64}
     ]
 for M in [M1, M2, M3]
     @testset "$(typeof(M))" begin
         for T in types
-            if M == M3 && T <: MMatrix
-                #TODO fix the issue that causes this failure
-                continue
+            exp_log_atol_multiplier = 8.0
+            if T <: MMatrix{3,3,Float64}
+                # eigendecomposition of 3x3 SPD matrices from StaticArrays is not very accurate
+                exp_log_atol_multiplier = 5.0e7
```

Comment:
`e7`? sounds like _really_ inaccurate.

Reply:
I guess `StaticArray` chooses fast over accurate but that's indeed quite extreme. The algorithm is good enough for `Float32` but for `Float64` one would expect something more accurate.